canvas_modules/common-canvas/src/common-properties/properties-controller.js

```diff
@@ -124,6 +124,22 @@ export default class PropertiesController {
 		return this.propertiesConfig;
 	}
 
+	setParamDef(paramDef) {
+		this.paramDef = paramDef;
+		const rightFlyout = (typeof this.getPropertiesConfig().rightFlyout !== "undefined")
+			? this.getPropertiesConfig().rightFlyout
+			: true;
+		const formData = Form.makeForm(paramDef, !rightFlyout);
+		this.setForm(formData);
+		if (formData) {
+			this.setTitle(formData.label);
+		}
+	}
+
+	getParamDef(paramDef) {
```

Comment:
I don't think this is needed.  The consuming application is passing that into us so they'll know what it is.